src/inausoft.netCLI/Deserialization/RegexOptionsDeserializer.cs

```diff
@@ -50,7 +50,9 @@ public object Deserialize(Type type, string optionsExpression)
 
                 if (property == null)
                 {
-                    throw new InvalidOptionException(optionName, $"Option {optionName} was not defined for {type}");
+                    var commandName = (Attribute.GetCustomAttribute(type, typeof(CommandAttribute)) as CommandAttribute).Name;
+
+                    throw new InvalidOptionException(commandName, optionName, $"Option {optionName} was not defined for {type}");
```

Comment:
Consider providing exception message that would make sense when variables were stripped out. I observed this interesting approach in most (all?) .NET libs and it's helpful when searching the web for solutions for exceptions.

Eg. when I see this exception I would like to google it, but without my custom values. In current situation I would end up in searching for "Option was not defined for".

I suggest updating exception message to `$"Option '{optionName}' is not defined for command '{type}'."`

Reply:
Agreed.